package.json

```diff
@@ -56,6 +56,7 @@
     "protractor": "~5.1.0",
     "ts-node": "~2.0.0",
     "tslint": "~4.4.2",
-    "typescript": "~2.0.0"
+    "typescript": "~2.0.0",
+    "webpack": "^2.3.1"
```

Comment:
I don't believe it's needed. I tried a couple different methods for configuring sass, and that was probably just something I forgot to remove. I'll update that as well as remove the compiled css file.